pkg/apis/kubeadm/bootstraptoken/v1/utils.go

```diff
@@ -24,13 +24,25 @@ import (
 
 	"github.com/pkg/errors"
 
-	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
Why is this being changed back to a worse alias?

Reply:
https://github.com/kubermatic/kubeone/pull/3358#discussion_r1727404220